scripts/vis_trawler.py

```diff
@@ -449,27 +449,28 @@ def s3_create_anon_access_policy(bucket_name):
     anon_access_policy: A json formatted s3 bucket policy
     """
     anon_policy_dict = {
-        "Version":"2012-10-17",
-        "Statement":[
+        "Version": "2012-10-17",
+        "Statement": [
             {
-            "Sid":"AddPerm",
-            "Effect":"Allow",
-            "Principal": "*",
-            "Action":["s3:GetObject"], #, "s3:ListBucket"],
-            "Resource":["arn:aws:s3:::%s/*" % bucket_name]
+                "Sid": "AddPerm",
+                "Effect": "Allow",
+                "Principal": "*",
+                "Action": ["s3:GetObject"],
```

Comment:
What is the reason for removing "s3:ListBucket" ?

Reply:
Just checking up on this comment the s3:ListBucket is in the definition below this one, defined at the bucket level. Is this what you're talking about?

Reply:
Didn't see the comment in the previous Action line (i.e. previously looked like s3:ListBucket was in twice) - ignore this query :)

Reply:
👍